docs/platform/howto/disk-autoscaler.md

```diff
@@ -7,6 +7,14 @@ import TabItem from '@theme/TabItem';
 import ConsoleLabel from "@site/src/components/ConsoleIcons"
 import {ConsoleIcon} from "@site/src/components/ConsoleIcons"
 
+<!-- vale off -->
+
+export const Example = () => (
+  <p>If you have a 500 GB plan and want to scale up to 1 TB using the autoscaler, the autoscaler value should be 1000 GB.</p>
```

Comment:
Our 500 GB is actually 500 * 1024 * 1024 * 1024 bytes so it would be more accurate to say 1024 GB.
(https://en.wikipedia.org/wiki/Byte#Multiple-byte_units)